gensim/models/ldamodel.py

```diff
@@ -48,7 +48,7 @@
 from gensim.models.callbacks import Callback
 
 # log(sum(exp(x))) that tries to avoid overflow
-from scipy.misc import logsumexp
+from scipy.special import logsumexp
```

Comment:
For `1.0.0` it works, but as you see in https://github.com/RaRe-Technologies/gensim/blob/develop/setup.py#L290 our minimal version is `0.18.1`, in this version, this import will be failed. 

Import **must** work for any scipy `>=0.18.1`